rest-service/manager_rest/rest/resources_v3_1/execution_schedules.py

```diff
@@ -0,0 +1,275 @@
+        rm = get_resource_manager()
+        deployment_id = request_dict['deployment_id']
+        deployment = rm.sm.get(models.Deployment, deployment_id)
+        rm._validate_permitted_to_execute_global_workflow(deployment)
+        rm._verify_workflow_in_deployment(workflow_id,
+                                          deployment,
+                                          deployment_id)
+
+        # --- Schedule event parameters
+        since = self._verify_timestamp(request_dict['since'])
+        until = self._verify_timestamp(request_dict.get('until'))
+
+        # we need to have at least: rrule; or count=1; or frequency
+        rrule_string = request_dict.get('rrule')
+        frequency = request_dict.get('frequency')
+        weekdays = request_dict.get('weekdays')
+        count = request_dict.get('count')
+
+        if rrule_string:
+            if frequency or weekdays or count:
+                raise manager_exceptions.BadParametersError(
+                    "`rrule` cannot be provided together with `frequency`, "
+                    "`weekdays` or `count`.")
+            try:
+                rrule.rrulestr(rrule_string)
+            except ValueError as e:
+                raise manager_exceptions.BadParametersError(
+                    "invalid RRULE string provided: {}".format(str(e)))
+            rule = {'rrule': rrule_string}
+        else:
+            if count:
+                count = convert_to_int(request_dict.get('count'))
+            frequency = self._verify_frequency(request_dict.get('frequency'))
+            weekdays = self._verify_weekdays(request_dict.get('weekdays'))
+            if not frequency and count != 1:
+                raise manager_exceptions.BadParametersError(
+                    "frequency must be specified for execution count larger "
+                    "than 1")
+            rule = {
+                'frequency': frequency,
+                'count': count,
+                'weekdays': weekdays
+            }
+
+        slip = request_dict['slip']
+        stop_on_fail = verify_and_convert_bool(
+            'stop_on_fail',  request_dict.get('stop_on_fail', False))
+
+        now = get_formatted_timestamp()
+        schedule = models.ExecutionSchedule(
+            id=schedule_id,
+            deployment=deployment,
+            created_at=now,
+            since=since,
+            until=until,
+            rule=rule,
+            slip=slip,
+            workflow_id=workflow_id,
+            parameters=parameters,
+            execution_arguments=execution_arguments,
+            stop_on_fail=stop_on_fail,
+        )
+        return rm.sm.put(schedule), 201
+
+    @rest_decorators.marshal_with(models.ExecutionSchedule)
+    @authorize('execution_start')
+    def patch(self, schedule_id, **kwargs):
+        """Updates scheduling parameters of an existing execution schedule"""
+
+        validate_inputs({'schedule_id': schedule_id})
+        request_dict = get_json_and_verify_params()
```

Comment:
not much point for either of those lines? the verify has no arguments, and the schedule_id, we look that up in the db anyway